projects/go-lib/src/lib/components/go-table/go-table-column.component.ts

```diff
@@ -11,6 +11,7 @@ export class GoTableColumnComponent extends GoTableChildColumnComponent {
   @Input() sortable?: boolean;
   @Input() title: string;
   @Input() width: number;
+  @Input() wrapContent: boolean = null;
```

Comment:
Would it be better to give this a default value of false? I'm not sure of the implications of doing it null v. false here but I feel like if the user didn't define the input then it would at least have a false fall-back value.

Reply:
Good question. I ended up going the `null` route here because I wanted the ability to tell if the implementor specifically set the attribute on the column and because the wrapping could go both ways.

For Ex:
```html
<go-table [wrapContent]="true">
  <go-table-column field="id"></go-table-column>
  <go-table-column field="name" [wrapContent]="false"></go-table-column>
</go-table>
```
The `id` column will be null so it will use whatever was specified on the `go-table` component. The `name` column, however, is no longer `null` and will use the value passed to `wrapContent`.

The opposite scenario is also true:
```html
<go-table [wrapContent]="false">
  <go-table-column field="id"></go-table-column>
  <go-table-column field="name" [wrapContent]="true"></go-table-column>
</go-table>
```

See the logic of this [here](https://github.com/mobi/goponents/pull/820/files#diff-68fef6945ac75f361cff6412124644568c3fd5392e70a16bd82a9cee8b4f3892R123).

Reply:
Ah gotcha, so the idea is that null allows it to inherit from its parent element. Cool!